classes/ActionScheduler_Abstract_QueueCleaner.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Abstract class with common Queue Cleaner functionality.
+ */
+abstract class ActionScheduler_Abstract_QueueCleaner {
```

Comment:
Was this accidentally given the `QueueCleaner` suffix?

It seems like `ActionScheduler_Abstract_QueueRunner` is a more suitable name as it includes the common `QueueRunner` functionality, not functionality from `ActionScheduler_QueueCleaner`.

Reply:
Yep, that was a mistake in naming. I'll change that to the correct name.